apis/apiextensions/v1/xrd_types.go

```diff
@@ -98,6 +98,14 @@ type CompositeResourceDefinitionVersion struct {
 	// Served specifies that this version should be served via REST APIs.
 	Served bool `json:"served"`
 
+	// The deprecated field specifies that this version is deprecated and should not be used.
+	// +optional
+	Deprecated *bool `json:"deprecated,omitempty"`
+
+	// DeprecationWarning specifies the message that should be shown to the user when using this version.
```

Comment:
Are these comments wrapped at 80 characters? If not could you make sure to wrap them! 🙂

Reply:
Fixed